macros/sql/get_column_values.sql

```diff
@@ -11,19 +10,45 @@ Returns:
     A list of distinct values for the specified columns
 #}
 
-{% macro get_column_values(table, column, max_records=none) -%}
+{% macro get_columns(table, column, max_records=none, fail=True, default='blank') -%}
+
+{#-- Prevent querying of db in parsing mode. This works because this macro does not create any new refs. #}
+    {%- if not execute -%}
+        {{ return('') }}
+    {% endif %}
+{#--  #}
+
+    {%- set table_value = adapter.get_relation(database=table.database,
+                                          schema=table.schema,
+                                         identifier=table.identifier) -%}
+
+    {%- call statement('get_column_values', fetch_result=true) %}
+
+        {%- if not table_value and fail -%}
+
+          {{ log("Table doesn't exist..") }}
```

Comment:
```suggestion
```

I don't think we need this log line, as the exception you have here will show up in the debug logs too